order.js

```diff
@@ -33,16 +33,120 @@ Render.main = (data) => {
 
     <h3 id="proofs">Proofs</h3>
     ${Render.try('proofs', data)}
+
+    <h3 id="messages">Message History</h3>
+    <div class="section">
+      ${Render.try('messages', data)}
+    </div>
   `;
   return result;
 };
 
+Render.messages = (data) => {//<small style="top: 0;" class="datetime">${item.created}</small>
+  var result = '';
+  data.order_messages.forEach((item) => {
+    var msgDate = new Date(item.created);
+    var day = msgDate.getDate();
+    switch(day) {
+      case 0: day = "00"; break; case 1: day = "01"; break; case 2: day = "02"; break; case 3: day = "03"; break; case 4: day = "04"; break; case 5: day = "05"; break; case 6: day = "06"; break; case 7: day = "07"; break; case 8: day = "08"; break; case 9: day = "09"; break;
+    }
+    var month = msgDate.getMonth();
+    switch(month) {
+      case 0: month = "Jan"; break; case 1: month = "Feb"; break; case 2: month = "Mar"; break; case 3: month = "Apr"; break; case 4: month = "May"; break; case 5: month = "Jun"; break; case 6: month = "Jul"; break; case 7: month = "Aug"; break; case 8: month = "Sep"; break; case 9: month = "Oct"; break; case 10: month = "Nov"; break; case 11: month = "Dec"; break;
+    }
+    var year = msgDate.getFullYear();
+    var time = solveTimeString(msgDate);
+
+    if(item.subject == '' || item.subject == null) {//no subject
+      if(item.from == 'You') {
```

Comment:
You'll want to do this kind of logic using css.
I'd suggest a class called card.message - with default styling
Then card.message.cfg - for styling related to from = CFG
card.message.you for styling from cust messages.

Then in the code body you only need to switch between the 2 classes instead of representing all the differences in code